services/api/src/v1/__openapi__/shops.json

```diff
@@ -0,0 +1,279 @@
+{
+  "objects": [
```

Comment:
Dont think this is an openapi json doc. 
objects is not valid => doesn't follow the spec. which might be fine but yeah wondering if worth calling it openapi if its not.